lib/arke/configuration.rb

```diff
@@ -9,11 +9,12 @@ def self.configure
   end
 
   class Configuration
-    attr_accessor :host, :api_key
+    attr_accessor :host, :api_key, :target, :sources, :strategy
 
     def initialize
       @host = ''
       @api_key = {}
+      @sources = []
```

Comment:
you forgot to add `target` and `strategy` here?

Reply:
I do not thing we need defaults there

Reply:
but does it add them later after you init config?

Reply:
Yeap, np, `@target={"driver"=>"rubykube", "name"=>"John", "key"=>nil, "secret"=>nil}`

Reply:
cool